contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -384,13 +405,13 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     // solhint-disable-next-line custom-errors
     require(msg.sender == address(s_link), "only callable from LINK");
 
-    (uint32 callbackGasLimit, uint16 requestConfirmations, uint32 numWords, bytes memory extraArgs) = abi.decode(
+    (uint32 callbackGasLimit, uint16 requestConfirmations, uint32 numWords, bytes memory extraArgs, bool isFeedStale) = abi.decode(
       _data,
-      (uint32, uint16, uint32, bytes)
+      (uint32, uint16, uint32, bytes, bool)
```

Comment:
the data here is set from the wrapper consumer contract and won't have the `isFeedStale` encoded in it right? 

Do we need to remove `isFeedStale` from here and initialize it below:

``` 
(int256 weiPerUnitLink, bool isFeedStale) = _getFeedData();
```

Reply:
Oh I see that `isFeedStale` is passed in to `transferAndCall` below in Wrapper consumer base. I don't think we need to pass in this `isFeedStale` extra data if we can retrieve it in `_getFeedData();` call anyways.

Reply:
Yes, because the concept of having a fallback wei comes from the wrapper and not the wrapper consumer base, so I felt that the wrapper is a better place to emit the event, with knowledge of the actual fallback value. Unless all we need is just a stale event that doesn't tell us what the fallback value used is.

The actual amount charged comes from the first `_getFeedData()` in `calculateRequestPrice`. The second `_getFeedData()` seems to be a safeguard just in case the price has changed. And I don't think we return the excess amount in the event that the request price is now cheaper right? So since the LINK amount that is transferred to the wrapper contract and hence charged comes from the first `_getFeedData()`, I felt that that's the better place to emit the event.